src/daft-connect/Cargo.toml

```diff
@@ -53,7 +54,8 @@ python = [
   "dep:daft-recordbatch",
   "dep:daft-context",
   "dep:daft-catalog",
-  "dep:daft-functions"
+  "dep:daft-functions",
+  "dep:daft-functions-utf8"
```

Comment:
nit: from a similar comment, re-exporting the split crate via `daft-functions::utf8` would reduce refactors and I don't believe we would need to import daft-functions-utf8 here if we are already pulling daft-functions, but there could be something I'm missing.